src/Aspire.Hosting.Keycloak/KeycloakContainerImageTags.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Keycloak;
+
+internal static class KeycloakContainerImageTags
+{
+    public const string Registry = "quay.io";
+    public const string Image = "keycloak/keycloak";
+    public const string Tag = "24.0.4";
```

Comment:
We've been trying to stick to major.minor -- looks like they have good tagging so consider changing this to `24.0`

Reply:
Sure thing, done!